lib/CSSStyleDeclaration.js

```diff
@@ -22,28 +20,21 @@ const {
   parsePropertyValue,
   prepareValue
 } = require("./parsers");
-const allExtraProperties = require("./utils/allExtraProperties");
-const { dashedToCamelCase } = require("./utils/camelize");
-const { getPropertyDescriptor } = require("./utils/propertyDescriptors");
 const { asciiLowercase } = require("./utils/strings");
 
 /**
  * @see https://drafts.csswg.org/cssom/#the-cssstyledeclaration-interface
  */
 class CSSStyleDeclaration {
   /**
-   * @param {Function} onChangeCallback
-   * @param {object} [opt]
-   * @param {object} [opt.context] - Window, Element or CSSRule.
+   * @param {object} globalObject - Window
+   * @param {object} opt - Options
+   * @param {object} opt.context - Element or CSSStyleRule
+   * @param {Function} opt.onChange - Callback when cssText change or property removed
    */
-  constructor(onChangeCallback, opt = {}) {
-    // Make constructor and internals non-enumerable.
+  constructor(globalObject = globalThis, opt = {}) {
+    // Make internals non-enumerable.
```

Comment:
This extra work should no longer be necessary in the new architecture. We can just set properties normally.